src/styles/card.css

```
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 198px;
+  background: var(--color-white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+  box-sizing: border-box;
+}
+  
+.card_img {
+  width: 97.5%;
```

just add styles from figma
```suggestion
  width: 160px;
  height: 134px;
```